alfred/Makefile

```diff
@@ -76,7 +76,7 @@ define Package/alfred/install
 	[ "x$(CONFIG_PACKAGE_ALFRED_VIS)" == "xy" ] && cp -fpR  $(PKG_BUILD_DIR)/vis/batadv-vis $(1)/usr/sbin/ ; true
 	[ "x$(CONFIG_PACKAGE_ALFRED_GPSD)" == "xy" ] && cp -fpR  $(PKG_BUILD_DIR)/gpsd/alfred-gpsd $(1)/usr/sbin/ ; true
 	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/alfred.init $(1)/etc/init.d/alfred
+	$(INSTALL_BIN) ./files/alfred.procd $(1)/etc/init.d/alfred
```

Comment:
Either I am blind or no one else is using this *.procd extension in OpenWrt. If you want to move/rename something then use the complete paths (as seen on the device) in alfred/files/. But this should most likely be placed in a separate commit

Reply:
Okay, renamed back to `alfred.init`